README.md

```diff
@@ -6,7 +6,7 @@ http://creativecommons.org/licenses/by-sa/4.0/
 ![Build Status](https://github.com/knatten/cppquiz/actions/workflows/ci.yml/badge.svg)
 
 # Requirements
-- Python 3.6 or 3.7
+- Python 3.8 / python 3.9 / python 3.10
```

Comment:
I suggest just saying "Python 3.8 or higher" here, then we don't need to keep it in sync with ci.yml.

Reply:
https://github.com/knatten/cppquiz/pull/279/commits/2a546f5d7e1b30659b38b2f9ecbec2bf4fd27df0